tempesta_fw/sched/tfw_sched_hash.c

```diff
@@ -45,22 +45,25 @@
 
 MODULE_AUTHOR(TFW_AUTHOR);
 MODULE_DESCRIPTION("Tempesta hash-based scheduler");
-MODULE_VERSION("0.2.1");
+MODULE_VERSION("0.3.0");
 MODULE_LICENSE("GPL");
 
 typedef struct {
-	TfwSrvConn	*srv_conn;
-	unsigned long	hash;
-} TfwConnHash;
+	size_t			conn_n;
+	TfwServer		*srv;
+	TfwSrvConn		*conn[TFW_SRV_MAX_CONN];
+	unsigned long		hash[TFW_SRV_MAX_CONN];
+} TfwHashSrv;
```

Comment:
Comment https://github.com/tempesta-tech/tempesta/pull/666/files#r94275356 is still not addressed. I saw your PR https://github.com/tempesta-tech/tempesta/pull/714/files, but it's somehow already closed.